src/isar/apis/robot_control/robot_controller.py

```diff
@@ -17,7 +18,13 @@ def __init__(
         self.logger = logging.getLogger("api")
 
     def generate_media_config(self):
-        return self.robot_utilities.generate_media_config()
+        media_config = self.robot_utilities.generate_media_config()
+        if media_config is None:
+            raise HTTPException(
+                status_code=204,
```

Comment:
Nothing wrong with this, but just to check: do we handle recieving an empty response in Flotilla?

Reply:
Yes, that is handled in this PR https://github.com/equinor/flotilla/pull/1967